### PR TITLE
PyTorch training code

### PR DESCRIPTION
**Pull request by szagoruyko**
reproduces exactly (sometimes even slightly better accuracy) CIFAR-10 and CIFAR-100 training

moved from https://github.com/pytorch/examples/pull/22

### CODE REVIEW

**Comment by ZhouYanzhao** [comment]
:+1: Great work! By the way, there are duplicate _"main.py"s_ in the last line of README.

**Comment by szagoruyko** [comment]
@ZhouYanzhao thanks, fixed!
